src/rebuild/srv.c

```diff
@@ -1230,34 +1231,35 @@ rebuild_task_complete_schedule(struct rebuild_task *task, struct ds_pool *pool,
 			return rc;
 		}
 
-		/* Schedule reclaim to clean up current op */
+		/* Schedule reclaim to clean up current op. Let's keep go ahead to retry even if
+		 * reclaim the current rebuilding fails.
+		 */
 		rc = ds_rebuild_schedule(pool, task->dst_map_ver, ++task->dst_rebuild_gen,
 					 rgt->rgt_stable_epoch, &task->dst_tgts, RB_OP_RECLAIM, 5);
 		if (rc)
-			return rc;
+			D_ERROR(DF_UUID "schedule reclaim fail: "DF_RC"\n",
+				DP_UUID(task->dst_pool_uuid), DP_RC(rc));
```

Comment:
My concern is that if current "reclaim" failed, then whether it is still necessary or not to retry the rebuild task.

Reply:
there is no other choice here, except retry, since you can not skip the pool map version rebuild anyway.  The current idea is that if rebuild keeps failing, it will keep retry until admin steps in.  Note: this is all in background anyway.